pkg/debuginfo/debuginfod_test.go

```diff
@@ -72,47 +73,50 @@ func TestHTTPDebugInfodClient_request(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			c := &HTTPDebugInfodClient{
-				logger:          log.NewNopLogger(),
-				timeoutDuration: tt.fields.timeoutDuration,
-				client: &http.Client{
-					// Recorder transport passed, so the recorded data from testdata/fixtures will ber passed.
-					// Use make go/test-clean to remove the recorded data.
-					Transport: r,
-				},
-			}
-			ctx, cancel := context.WithTimeout(context.Background(), tt.fields.timeoutDuration)
-			t.Cleanup(cancel)
+		arch := runtime.GOARCH
```

Comment:
Everything should work with all the architecture, I think. Why do we have this?

Reply:
The want path here https://github.com/parca-dev/parca/pull/1060/commits/22106b0e35cdc9be5b7610b124ceef81bebe86b3#diff-6299d5e6bee95423b1ed778126b7433378b37d6a60daf47d5f332d5c2e33af1bR71 has ELF file in x86-64 format. So had to do it like this.